packages/mdl-drawer/util.js

```diff
@@ -66,18 +67,42 @@ function getTransformPropertyName(globalObj = window) {
  * @param {Object} globalObj The global object, useful for testing (defaults to window).
  * @return {boolean} Whether the current browser supports CSS properties.
  */
-function supportsCssCustomProperties(globalObj = window) {
+export function supportsCssCustomProperties(globalObj = window) {
   if ('CSS' in globalObj) {
     return globalObj.CSS.supports('(--color: red)');
   }
   return false;
 }
 
 /**
+ * Determine whether the current browser supports passive event listeners, and if so, use them.
+ * @param {Object} globalObj The global object, useful for testing (defaults to window).
+ * @return {Object|boolean} The third argument to the addEventListener method: either an options object or a boolean.
+ */
+export function applyPassive(globalObj = window) {
+  if (supportsPassive_ !== undefined && globalObj === window) {
```

Comment:
It may be easier for testing purposes to - instead of relying on `globalObj` being `window` - to just have a `forceRefresh = false` param that we use to re-perform the check. We can use this in testing. See https://github.com/google/material-design-lite/blob/master/packages/mdl-menu/util.js#L20